lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1629,6 +1629,7 @@ protected function _validateAndCompleteOneToOneMapping(array $mapping)
                     ? $joinColumn['fieldName']
                     : $joinColumn['name'];
             }
+            unset($joinColumn);
```

Comment:
It's just an additional instruction (also to read - not worried about performance)